tests/integration/test_retraining_policies.py

```diff
@@ -0,0 +1,194 @@
+    with cleanup_dr("datasets/", id_attribute="datasetId"):
+        yield get_or_create_dataset_from_df(
+            dr_endpoint,
+            dr_token,
+            use_case_id=use_case,
+            name="pytest_test_autopilot_dataset",
+            data_frame=df,
+        )
+
+
+@pytest.fixture
+def autopilot_model(dr_endpoint, dr_token, use_case, dataset, analyze_and_model_config, cleanup_dr):
+    with cleanup_dr("projects/", paginated=False):
+        yield get_or_create_autopilot_run(
+            dr_endpoint,
+            dr_token,
+            "pytest autopilot",
+            dataset,
+            analyze_and_model_config=analyze_and_model_config,
+            use_case=use_case,
+        )
+
+
+@pytest.fixture()
+def recommended_model(autopilot_model):
+    return dr.ModelRecommendation.get(autopilot_model).model_id
+
+
+@pytest.fixture
+def registered_model_name(dr_token_hash):
+    return f"pytest datarobot registered model {dr_token_hash}"
+
+
+@pytest.fixture
+def deployment_name(dr_token_hash):
+    return "pytest datarobot deployment #{i}"
+
+
+@pytest.fixture
+def registered_model_version(
+    cleanup_dr, dr_endpoint, dr_token, recommended_model, registered_model_name
+):
+    with cleanup_dr("registeredModels/"):
+        yield get_or_create_registered_leaderboard_model_version(
+            dr_endpoint, dr_token, recommended_model, registered_model_name
+        )
+
+
+@pytest.fixture
+def deployment_id(
+    cleanup_dr,
+    registered_model_version,
+    registered_model_name,
+    default_prediction_server_id,
+    dr_endpoint,
+    dr_token,
+):
+    with cleanup_dr("deployments/"):
+        yield get_or_create_deployment_from_registered_model_version(
+            endpoint=dr_endpoint,
+            token=dr_token,
+            registered_model_version_id=registered_model_version,
+            label=registered_model_name,
+            default_prediction_server_id=default_prediction_server_id,
+        )
+
+
+def test_retraining_policy(dr_endpoint, dr_token, deployment_id, dataset):
+    print(deployment_id)
+    # Can we create policy
+    retraining_id_1 = get_update_or_create_retraining_policy(
+        endpoint=dr_endpoint,
+        token=dr_token,
+        deployment_id=deployment_id,
+        name="post_test",
+        dataset_id=dataset,
+    )
+
+    # If we "update" policy, is the ID the same?
+    retraining_id_2 = get_update_or_create_retraining_policy(
+        endpoint=dr_endpoint,
+        token=dr_token,
+        deployment_id=deployment_id,
+        name="post_test",
+        dataset_id=dataset,
+    )
+    assert retraining_id_1 == retraining_id_2
+
+    # If we actually change a parameter, is the ID the same?
+    retraining_id_3 = get_update_or_create_retraining_policy(
+        endpoint=dr_endpoint,
+        token=dr_token,
+        deployment_id=deployment_id,
+        name="patch_test",
+        dataset_id=dataset,
+        featureListStrategy="informative_features",
```

Comment:
This isn't what gets set by default is it?

Reply:
Also noting that if you just add the featureListStategy to `retraining_id_2` you could first do `assert retraining_id_1 == retraining_id_2` and then do your get request and `assert policy_data["featureListStrategy"] == "informative_features"`